deluge/ui/console/widgets/popup.py

```diff
@@ -341,6 +343,10 @@ def __init__(
         width_req=0.5,
         **kwargs
     ):
+        if not isinstance(message, six.string_types):
+            log.warn("Expected string type for message. Got '%s'", type(message))
```

Comment:
`warn` has been deprecated: https://docs.python.org/3/library/logging.html#logging.warning

Reply:
I also don't think this check should be here and instead the decoding should be done in `_on_connect_fail`